### PR TITLE
MISUV-2910 Update Home page

### PR DESCRIPTION
**Pull request by andriikondiuk**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -68,20 +68,22 @@ home.unique.taxpayer.reference                                  = Unique Taxpaye
...
+home.paymentHistoryAndCredit.heading                            = Payment history and credit
+home.paymentHistoryAndCredit.view                               = Payment and refund history
 home.overdue.date.payment.count                                 = {0} OVERDUE PAYMENTS
 home.overdue.date.update.count                                  = {0} OVERDUE UPDATES
 home.overdue.date                                               = OVERDUE
-home.tax-years.heading                                          = Tax years
-home.tax-years.view                                             = View tax years
+home.tax-years.heading                                          = Returns
+home.tax-years.view                                             = View all tax years
 home.tax-years.viewPayments                                     = View payment history
```
Comment (stephenkward) [inline conf/messages:81]: @andriikondiuk - I don't think this is being used anymore.

File: conf/messages [hunk 2]
```diff
@@ -68,20 +68,22 @@ home.unique.taxpayer.reference                                  = Unique Taxpaye
...
-home.manage-income-tax.heading                                  = Manage Income Tax
 home.your-returns.heading                                       = Manage Income Tax
 home.your-returns.description                                   = Use this service to report untaxed income and any tax relief you’re claiming
 home.your-returns.agent.description                             = Use this service to report untaxed income and any tax relief you’re claiming.
-home.your-returns.updatesLink                                   = Update and submit an Income Tax Return
+home.your-returns.updatesLink                                   = Update and submit your {0} to {1} return
+home.returns.viewLink                                           = View your current {0} to {1} return
 home.update-and-submit-your-returns.Link                        = Update and submit an Income Tax Return
```
Comment (stephenkward) [inline conf/messages:87]: @andriikondiuk - I don't think this is being used anymore.

File: conf/messages [hunk 3]
```diff
@@ -68,20 +68,22 @@ home.unique.taxpayer.reference                                  = Unique Taxpaye
...
 home.self-assessment.updatesLink                                = View your Self Assessment tax years
```
Comment (stephenkward) [inline conf/messages:88]: @andriikondiuk - I don't think this is being used anymore.

File: conf/messages [hunk 4]
```diff
@@ -68,20 +68,22 @@ home.unique.taxpayer.reference                                  = Unique Taxpaye
...
 home.view-self-assessment.Link                                  = View your Self Assessment tax years
```
Comment (stephenkward) [inline conf/messages:89]: @andriikondiuk - I don't think this is being used anymore.

**Review by stephenkward — CHANGES_REQUESTED** [review]

File: conf/messages (hunk 4 above)
Comment (andriikondiuk) [reply conf/messages]: Done, thanks

File: conf/messages (hunk 3 above)
Comment (andriikondiuk) [reply conf/messages]: Done, thanks

File: conf/messages (hunk 2 above)
Comment (andriikondiuk) [reply conf/messages]: Done, thanks

File: conf/messages (hunk 1 above)
Comment (andriikondiuk) [reply conf/messages]: Done, thanks

**Review by stephenkward — APPROVED** [review]

**Review by stephenkward — APPROVED** [review]

**Pull request merged**